msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/DiscussionForumTool.java

```diff
@@ -5578,8 +5727,15 @@ public String processAddCommentToDeniedMsg()
 	  StringBuilder sb = new StringBuilder();
 	  sb.append("<div class=\"messageCommentWrap\">");
 	  sb.append("<div class=\"messageCommentMD\">");
-	  sb.append(getResourceBundleString(MOD_COMMENT_TEXT) + " ");
-	  sb.append(UserDirectoryService.getCurrentUser().getDisplayName());
+	  if (selectedTopic.getTopic().getPostAnonymous())
+	  {
+	  	sb.append(getResourceBundleString(MOD_COMMENT_TEXT_ANON));
+	  }
+	  else
+	  {
+	  	sb.append(getResourceBundleString(MOD_COMMENT_TEXT)).append(" ");
+	  	sb.append(UserDirectoryService.getCurrentUser().getDisplayName());
```

Comment:
I kinda like it like this vertical actually, but I've refactored a few other ones